src/malcolm/middleware/malcolmReduxMiddleware.js

```diff
@@ -21,9 +21,10 @@ function getNextId(malcolmState) {
 }
 
 function subscriptionActive(path, messagesInFlight) {
-  return messagesInFlight.some(
+  return Object.keys(messagesInFlight).some(
```

Comment:
I think you can use `Object.entries` here and then you don't have to do the look up